### PR TITLE
Bug 1209961 - Add BiDi icons to gaia-icons

### PR DESCRIPTION
**Pull request by autra**
This adds a data-icon that represents a BiDi-proof icon for each left/right pair
of icons that needs to be mirrored in RTL.
Ex: We generate 2 [data-icon="expand"] rules, one for LTR and one for LTR, that
selects respectively 'expand-left' and 'expand-right'.

### CODE REVIEW

File: Gruntfile.js [hunk 1]
```diff
@@ -41,6 +41,22 @@ module.exports = function(grunt) {
             mixinPrefix: ''
           }
         }
+      },
+
+      // create a helper css file for BiDi-proof icons
+      bidi: {
```
Comment (autra) [inline Gruntfile.js]: Here I feel like I'm abusing grunt-webfont... Do you see another better way of doing this?

Comment (wilsonpage) [inline Gruntfile.js]: I would just make and maintain the file by hand. There aren't going to be that many bdi icons.

**Comment by autra** [comment]
@wilsonpage PR updated :-)

File: bidi-helper.css [hunk 2]
```diff
@@ -0,0 +1,58 @@
+/* Some BiDi-proof icons */
+
+[data-icon="back"]:-moz-dir(ltr):before {
+  content: 'left';
+}
```
Comment (wilsonpage) [inline bidi-helper.css]: Nit: Line break between rules :)

Comment (autra) [inline bidi-helper.css]: In gaia, we don't skip line between ltr and rtl part of the same rule (AFAIK)o. What do you prefer here?

**Pull request merged**